tests/test_diffeq/test_wrappedscipysolver.py

```diff
@@ -0,0 +1,179 @@
+import numpy as np
+import pytest
+import pytest_cases
+from scipy.integrate._ivp import base, rk
+from scipy.integrate._ivp.common import OdeSolution
+
+from probnum import diffeq, randvars
+from probnum.diffeq import odesolution, wrappedscipyodesolution, wrappedscipysolver
+
+
+def case_lorenz():
+    y0 = np.array([0.0, 1.0, 1.05])
+    ode = diffeq.lorenz([0.0, 1.0], y0)
+    scipysolver = rk.RK45(ode.rhs, ode.t0, y0, ode.tmax)
+    testsolver = wrappedscipysolver.WrappedScipyRungeKutta(
+        rk.RK45(ode.rhs, ode.t0, y0, ode.tmax), order=4
+    )
+    return testsolver, scipysolver
+
+
+def case_logistic():
+    y0 = np.array([0.1])
+    ode = diffeq.logistic([0.0, 1.0], y0)
+    scipysolver = rk.RK45(ode.rhs, ode.t0, y0, ode.tmax)
+    testsolver = wrappedscipysolver.WrappedScipyRungeKutta(
+        rk.RK45(ode.rhs, ode.t0, y0, ode.tmax), order=4
+    )
+    return testsolver, scipysolver
+
+
+def case_lotkavolterra():
+    y0 = np.array([0.1, 0.1])
+    ode = diffeq.lotkavolterra([0.0, 1.0], y0)
+    scipysolver = rk.RK45(ode.rhs, ode.t0, y0, ode.tmax)
+    testsolver = wrappedscipysolver.WrappedScipyRungeKutta(
+        rk.RK45(ode.rhs, ode.t0, y0, ode.tmax), order=4
+    )
+    return testsolver, scipysolver
+
+
+@pytest.fixture
+def y():
+    return np.array([0.1])
+
+
+@pytest.fixture
+def times():
+    return [0.0, 1.0]
+
+
+@pytest.fixture
+def start_point():
+    return 0.5
+
+
+@pytest.fixture
+def stop_point():
+    return 0.6
+
+
+@pytest.fixture
+def dense_output():
+    return [base.DenseOutput(0, 1)]
+
+
+@pytest.fixture
+def lst():
```

Comment:
can you maybe rename this to something that is easier to read? It can be super long, if necessary, but `lst` is a bit cryptic :)